training/distributed_training/tensorflow/data_parallel/efficientnet/EfficientNet_Notebook.ipynb

```diff
@@ -0,0 +1,368 @@
+{
```

Comment:
<p>Provide an IAM managed policy name for FSx. For instance, <code>AmazonFSxFullAccess</code>  .</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3060/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>done</p>